website/static/js/pages/statistics-page.js

```diff
@@ -13,3 +19,78 @@ $(function(){
         $adBlock.show();
     }
 });
+
+keenAnalysis.ready(function(){
+    function drawAllCharts(statsStartDate, statsEndDate) {
+        projectUsageStats.visitsByDay('#visits', startDate, endDate);
+        projectUsageStats.topReferrers('#topReferrers', startDate, endDate);
+        projectUsageStats.popularPages('#popularPages', startDate, endDate);
+        projectUsageStats.visitsServerTime('#serverTimeVisits', startDate, endDate);
+    }
+
+    function updateStartDate() {
+        startPicker.setStartRange(startDate);
```

Comment:
Minor: It's a bit confusing to use variables that are outside the scope of this function (`startPicker` and `startData`). In contrast, `drawAllCharts` receives its dependencies as arguments. Better to make this consistent. Alternatively, you could create a class that stores the dependencies as attributes.

```js
var DatePicker = oop.defclass({
    constructor: function(elem) {
        var self = this;
        self.startDate = ...
        self._startPicker = new pikaday(..)
    },
    updateStartDate: function(start) {
        self._startPicker.setStartRange(start)
        ...
    }
})
```

Certainly not a blocker, though.